test/exec/child.js

```diff
@@ -1,3 +1,5 @@
 // does nothing child process
 console.log("Child process.id: "+process.pid);
 console.log(" - - - - - - - - - - - - - - - - - - - - - - - ");
+// keep process alive for one minute
+setTimeout(function () {}, 60000);
```

Comment:
do you want to go ahead and merge this, or do you want me to implement polling?
Polling does have one drawback, I realized.  It's unlikely but possible that a new process starts with the same pid as one that was killed, causing terminate to poll forever.
I have implemented polling in [my crater-util package](https://github.com/jedwards1211/crater-util/blob/master/src/kill.js) (ironically I ended up using that instead of this) and it works well enough for my integration tests though.